### DISCUSSION TITLE
Server Ip

### DISCUSSION THREAD

**Discussion by Dark-LYNN** [community/community#46649] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

So i made a discord bot and use github pages as a discord bot website,
and to display the status and stats of the discord bot i use express but to have a be more secure i added so only a single IP could request the data from that port, but i cant find what the ip of the github.io is. where can i find this information?

**Comment by airtower-luna** [comment]
Check the ["meta" API endpoint](https://docs.github.com/en/rest/meta?apiVersion=2022-11-28#get-github-meta-information). I assume the `pages` field of the response should answer your question. Mind that the list might change now and then. However that probably won't help for your situation: Pages doesn't support any server-side code, so requests caused by visiting that website will come from the visitors' browsers.
